packages/upload/src/FzUpload.vue

```diff
@@ -68,12 +68,33 @@ const props = withDefaults(defineProps<FzUploadProps>(), {
   buttonLabel: "Carica",
   dragAndDropLabel: "o trascina qui",
 });
-const emit = defineEmits([]);
-const model = defineModel<File[]>();
+const emit = defineEmits(["fzupload:change"]);
+const model = defineModel<File[]>({
+  default: [],
+});
 
 const input = ref<HTMLInputElement | null>(null);
 const urlByFileMap = ref(new Map<File, string>());
 
+onMounted(() => {
+  if (model.value && model.value.length > 0 && !props.multiple) {
+    console.warn(
+      "[Fiscozen Design System]: FzUpload prop 'multiple' is set to false, but multiple files are provided. Only the first file will be used.",
+    );
```

Comment:
this warning could also go in the setter, even more so considering you're only checking at mount and never after that